wwwroot/js/adyenImplementation.js

```diff
@@ -1,16 +1,44 @@
 const clientKey = document.getElementById("clientKey").innerHTML;
-const type = document.getElementById("type").innerHTML;
 
-async function initCheckout() {
+// Used to finalize a checkout call in case of redirect
+const urlParams = new URLSearchParams(window.location.search);
+const sessionId = urlParams.get('sessionId'); // Unique identifier for the payment session
+const redirectResult = urlParams.get('redirectResult');
+
+// Typical checkout experience
+async function startCheckout() {
+  // Used in the demo to know which type of checkout was chosen
+  const type = document.getElementById("type").innerHTML;
+
+  try {
+    const checkoutSessionResponse = await callServer("/api/sessions?type=" + type);
```

Comment:
Was the `type` parameter supposed to be used for something when the session is created? It looks like it is ignored by the API controller.

Reply:
This is a remnant from before sessions where type was being used for some cases, but indeed as it is now it is not used at all. I'll remove it